api/auth/auth.go

```diff
@@ -86,33 +87,51 @@ func Login(ctx context.Context) error {
 		}
 	}()
 
+	var token *oauth2.Token
+
 	// Generate random state
 	b := make([]byte, 32)
 	rand.Read(b)
 	state := hex.EncodeToString(b)
 
 	tokenURL := authConfig.AuthCodeURL(state)
-	lg.Info().Msgf("opening %s", tokenURL)
-	if err := browser.OpenURL(tokenURL); err != nil {
-		return err
-	}
+	lg.Info().Msgf("attempting to open %s", tokenURL)
+	if err := browser.OpenURL(tokenURL); true {
```

Comment:
Doesn't the `true` mean that it will always choose this path? I'm guessing that's not what we want all of the time.

Reply:
:laughing: forgot to revert this. good catch